app/models/hint.rb

```diff
@@ -1,5 +1,36 @@
+require 'stringex/core_ext'
```

Comment:
Hmmm. I guess not. I just had some unlucky failed tests (I removed it locally then ran a few test runs and they all failed... but that was just a string of unlucky flakey tests. It works fine removed.)